asabiris/orchestration/kafkahandler.py

```diff
@@ -80,29 +80,30 @@ async def consume(self):
 			except Exception:
 				L.exception("General error when dispatching message")
 
-
 	async def dispatch(self, msg):
-		msg_type = msg.pop("type", "<missing>")
+		msg_type = msg.get("type", "<missing>")
```

Comment:
`.pop` should be correct, why `.get()`?

Reply:
@ateska Because we are popiing type before validation. And type is part of validation 

`required": ["type", "body"],`

Reply:
Ok. then you should not pop() that in each brach.